README.md

```diff
@@ -16,17 +16,17 @@ A tool to generate prioritized genes using phenotype information.
 ## Quickstart
 
 * Download [vibe .jar file][vibe_download]
-* Download and extract [TDB][tdb_download]
+* Download and extract [HDT][hdt_download]
 * Download the [HPO.owl][hpo_owl]
 * Make sure you have [Java 8 or higher][java_download]
-* Open a terminal and run VIBE. `java -jar vibe-with-dependencies.jar -d -t TDB/ -o results.tsv -p HP:0002996 -p HP:0001377`
+* Open a terminal and run VIBE. `java -jar vibe-with-dependencies-<version>.jar -d -t vibe-<db-version>.hdt -o results.tsv -p HP:0002996 -p HP:0001377`
```

Comment:
when I try to run with these arguments, I get an error:  Missing arguments: -w

when adding the missing argument it works fine

Reply:
seems like a documentation error that already existed before this PR